config/config.php

```diff
@@ -166,6 +166,12 @@ functionality to PhpReport
  */
 define('ALL_USERS_GROUP', 'staff');
 
+/**
+ * @name EMPLOYEES_GROUP
+ * @global string users group used for retrieving all active employees
+ */
+define('EMPLOYEES_GROUP', 'employee');
```

Comment:
I would prefer that the default value was 'staff', because 'employee' does not match any of the groups in a default install. It's a bit annoying for development purposes.

Reply:
I mean, what's annoying is that the "Team calendar" in vacation management would not work because no users would be listed.

Reply:
ok no problema, I just changed to 'staff'